src/directory/__tests__/getMetaStringObj.test.js

```diff
@@ -0,0 +1,39 @@
+import { getMetaStringObj } from '../getMetaStringObj.mjs';
+import JSON5 from 'json5';
+
+describe('getMetaStringObj', () => {
+  it('should return meta object for a valid page file path with valid meta object', async () => {
+    const filePath = './src/directory/__tests__/helpers/valid-meta.mdx';
+    const expectedMetaObject = JSON5.parse(
+      `{"title":"Build & connect backend","description":"Learn more about how you can build the backend for your app or connect with existing resources.","platforms":["android","angular","flutter","javascript","nextjs","react","react-native","swift","vue"],"route":"/[platform]/build-a-backend","canonicalObjects":[{"platforms":["vue","javascript","react","angular","nextjs"],"canonicalPath":"/javascript/build-a-backend/"}]}`
+    );
+
+    const results = await getMetaStringObj(filePath);
+
+    expect(results).toEqual(expectedMetaObject);
+  });
+
+  it('should throw an error for page with missing meta object', async () => {
+    const filePath = './src/directory/__tests__/helpers/missing-meta.mdx';
```

Comment:
I'm passing in a test mdx file for these tests, is there a reason I should mock .readFile instead?